js/apps/admin-ui/src/client-scopes/details/ScopeForm.tsx

```diff
@@ -190,13 +190,9 @@ export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
           min={0}
         />
         <ActionGroup>
-          <Button
-            variant="primary"
-            type="submit"
-            isDisabled={!isDirty || !isValid}
-          >
+          <FormSubmitButton formState={formState} variant="primary">
```

Comment:
Are there cases where the `FormSubmitButton` isn't `primary`? Perhaps we can make this the default and allow it to be overwritten where needed.

Reply:
It appears that React's `<Button>` have a default type of `primary`. 

```
# Button.tsx
const ButtonBase: React.FunctionComponent<ButtonProps> = ({
  ...
  variant = ButtonVariant.primary,
  iconPosition = 'start',
  'aria-label': ariaLabel = null,
  icon = null,
  ouiaId,
  ouiaSafe = true,
  tabIndex = null,
  innerRef,
  countOptions,
  ...props
}: ButtonProps) => {
  ...
}
```

Anyways, I have explicitly specified the `primary` type in `FormSubmitButton` and allow it to be override-able by placing `{...rest}` after `variant`
```
export const FormSubmitButton = ({
  ...
  ...rest
}: PropsWithChildren<FormSubmitButtonProps>) => {
  return (
    <Button
      variant="primary"
      ...
      {...rest}
      type="submit"
    >
      {children}
    </Button>
  );
};
```